app/src/main/java/com/taskruler/RetrofitClientInstance.kt

```diff
@@ -9,14 +9,14 @@ import retrofit2.converter.gson.GsonConverterFactory
  */
 object RetrofitClientInstance {
     private var retrofit: Retrofit? = null
-    private val BASE_URL = "https://wrytheruc.github.io/"
+    private const val BASE_URL = "https://wrytheruc.github.io/"
```

Comment:
turned the val to a const val to follow best practices.

cite: https://kotlinlang.org/docs/coding-conventions.html#overload-layout